emails/tests/fixtures/plain_text_expected.email

```diff
@@ -0,0 +1,179 @@
+Subject: Text-Only Email
+From: "root@server.example.com [via Relay]" <ebsbdsan7@test.com>
+To: user@example.com
+Reply-To: replies@default.com
+Resent-From: root@server.example.com
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="==[BOUNDARY0]=="
+
+--==[BOUNDARY0]==
+Content-Type: multipart/alternative;
+ boundary="==[BOUNDARY1]=="
+MIME-Version: 1.0
+
+--==[BOUNDARY1]==
+Content-Type: text/plain; charset="us-ascii"
+MIME-Version: 1.0
+Content-Transfer-Encoding: 7bit
+
+This email was sent to your alias ebsbdsan7@test.com. To stop receiving emails sent to this alias, update the forwarding settings in your dashboard.
+---Begin Email---
+This is a text-only email.
+See https://text.example.com
+
+--==[BOUNDARY1]==
+Content-Type: text/html; charset="us-ascii"
```

Comment:
comment (non-blocking): This seems strange, but IIRC we intentionally "enhance" text-only emails with an additional `text/html` part so we can put our rich header and footer into them.